BrandonBeltran/hackerrank/03/JavaList.java

```diff
@@ -0,0 +1,36 @@
+package bootcamp;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class JavaList {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int size = scanner.nextInt();
+        List <Integer> numbers = new ArrayList<>();
+
+        for(int iterator = 0; iterator < size; iterator++){
+            numbers.add(scanner.nextInt());
+        }
+
+        int queries = scanner.nextInt();
+
+        for(int iterator = 0; iterator < queries; iterator++){
+            String operation = scanner.next();
+            int position = scanner.nextInt();
+
+            if(operation.equals("Insert")){
+                int addition = scanner.nextInt();
+                numbers.add(position, addition);
+            }else if (operation.equals("Delete")){
```

Comment:
What if position > number.size() when trying numbers.remove() ???

Reply:
I'd throw an IndexOutOfBoundsException!! I'll fix that, thank you :)